includes/class-wc-payments.php

```diff
@@ -1633,6 +1636,19 @@ public static function init_woopay() {
 		}
 	}
 
+	/**
+	 * Load stripe site messaging script.
+	 *
+	 * @return void
+	 */
+	public static function load_stripe_site_messaging() {
```

Comment:
Could we reflect somehow in naming it's BNPLs related?

Reply:
Done here [55c6ff2](https://github.com/Automattic/woocommerce-payments/pull/6522/commits/55c6ff25ddbeb78cc19c452d3aacf33eb1fbdefd) thanks!